custom/icds_reports/tasks.py

```diff
@@ -1209,6 +1212,211 @@ def build_incentive_report(agg_date=None):
                 build_incentive_files.delay(location, agg_date, file_format, 3, location.parent.parent, location.parent)
 
 
+def aggregate_validation_helper(agg_date):
+    """
+    Validates the performance reports vs aggregate reports and send mails with mismatches
+    """
+    awc_performance_rows_list = list(AWWIncentiveReport.objects.filter(month=agg_date)
```

Comment:
not blocking but these don't need to be coerced to a list. you should be able to just iterate right over them, and it is probably more efficient that way